core/primitives/src/ed25519.rs

```diff
@@ -53,7 +53,9 @@ pub struct Pair(ed25519_dalek::Keypair);
 impl Clone for Pair {
 	fn clone(&self) -> Self {
 		Pair(ed25519_dalek::Keypair {
-			public: self.0.public.clone(),
+			// No `clone()` required here because `self.0.public: PublicKey`
+			// implements `Copy.`
```

Comment:
remove comment

Reply:
I planned to have some comments where there may be doubt over what was done.
I'll remove them, as requested.